src/core.js

```diff
@@ -1159,6 +1159,7 @@ define([
                 }
                 hidePageSpinner();
             } catch (e) {
+                window.console.log(util.formatExc(e));
```

Comment:
unsure if you meant to keep this in here, although I don't think it's a bad idea to have